### ISSUE TITLE
Icons Missing and Font Wrong

### ISSUE THREAD

**Issue by DaveH1** [Alex313031/Thorium-Win#210]
This site -
https://forums.digitalspy.com
is showing in an incorrect font in Thorium, and also has icons missing at the top and bottom of its pages.
It works fine in 360Chrome and Supermium.
Could this be looked at please?
This problem does exist elsewhere too, but this site is a good example.
Thanks, Dave.

**Comment by Unknown78** [comment]
It's fine in my system.

What's your OS and version?
Which build that you're running from? v123 avx2?

**Comment by DaveH1** [comment]
I'm using the 32 bit version on Windows XP SP3.
Thorium 122.0.6261.168.
If this has been fixed in version 123, I will certainly try it!
Thanks, Dave.

**Comment by Unknown78** [comment]
You're in the wrong repository, this issue should be posted on `thorium-legacy`.
https://github.com/Alex313031/thorium-legacy/issues

**Comment by Alex313031** [comment]
@DaveH1 Yeah @Unknown78 Is right, you are in the wrong repo. However, I have fixed GDI fonts being askew by rolling a newer version of Skia and then using a patch that win32ss created. New XP-only builds will be coming soon (like today or tommorow), that should fix text alignment.

**Comment by DaveH1** [comment]
Thank you so much, @Unknown78 and @Alex313031.
Sorry for posting in the wrong place, I'll know where to ask next time!
Looking forward to the new build, and thank you so much for still supporting XP.
I use Windows 10 as well on my multi-boot machine, but XP is still my OS of choice, and it's really great to have a browser based on a recent Chromium version to use on it. Kudos to you, and of course to win32ss as well!
Cheers, Dave.
